modules/learn/pages/data/distributed-transactions.adoc

```diff
@@ -0,0 +1,315 @@
+= Transactions
+:description: pass:q[A _transaction_ is an atomic unit of work that contains one or more operations. It is a group of operations that are either committed to the database together or they are all undone from the database.]
+:page-aliases: acid-transactions,transactions,learn:data/distributed-acid-transactions,introduction:distributed-acid-transactions,learn:data/transactions
+:tabs:
+
+[abstract]
+{description}
+
+== Overview
+
+Couchbase transactions support ACID properties for protected actions on the database.
+
+*Atomicity* ensures that a transaction provides all-or-nothing semantics i.e.  either all the documents modified in a transaction are committed, or none of the changes are committed. If there is a failure during transaction execution (such as the client crashing) all its changes are rolled back. 
```

Comment:
I'd put an em dash before the i.e., but a semi-colon or even a comma would be fine.
Just a nitpick, certainly not a blocker.